utils/config/examples/daos_server_unittests.yml

```diff
@@ -0,0 +1,69 @@
+# Example configuration file using sockets
+
+name: daos_server           # map to -g daos_server
+port: 10001                 # mgmt port
+provider: ofi+sockets       # map to CRT_PHY_ADDR_STR=ofi+sockets
+socket_dir: /tmp/daos_sockets
+control_log_mask: DEBUG
+control_log_file: /tmp/daos_control.log
+
+# single server instance per config file for now
+servers:
+-
+  targets: [0-0]            # map to -t 1, just count number of cpus in the range for now
+  fabric_iface: eth0        # map to OFI_INTERFACE=lo
+  log_mask: DEBUG,RPC=ERR,MEM=ERR
+  log_file: /tmp/server.log # map to D_LOG_FILE=/tmp/server.log
+  env_vars:                 # influence DAOS IO Server behaviour by setting env variables
+  - DAOS_MD_CAP=1024
+  - CRT_CTX_SHARE_ADDR=0
+  - CRT_TIMEOUT=30
+  - FI_SOCKETS_MAX_CONN_RETRY=1
+  - FI_SOCKETS_CONN_TIMEOUT=2000
+  - DD_SUBSYS=all
+  - DD_MASK=all
```

Comment:
this should be daos_default rather than all

Reply:
though I guess this only affects rdb test so maybe it's fine

Reply:
as above